Sources/Subs-Db-postgresql.php

```diff
@@ -487,13 +487,10 @@ function smf_db_affected_rows($result = null)
  */
 function smf_db_insert_id($table, $field = null, $connection = null)
 {
-	global $db_connection, $smcFunc, $db_prefix;
+	global $smcFunc, $db_prefix;
 
 	$table = str_replace('{db_prefix}', $db_prefix, $table);
 
```

Comment:
Is there a reason for removing this check? $connection in this case is a DB connection resource. It could possibly be false if the connection failed, in which case defaulting to $db_connection is probably wise.